cmd/influx/main.go

```diff
@@ -185,26 +186,72 @@ func newApiClient(ctx *cli.Context, cli *internal.CLI, injectToken bool) (*api.A
 	return api.NewAPIClient(apiConfig), nil
 }
 
-// standardCtx returns a context that will cancel on SIGINT and SIGTERM.
-func standardCtx(ctx *cli.Context) context.Context {
-	return signals.WithStandardSignals(ctx.Context)
-}
-
```

Comment:
This is eliminated as we use the `cli.App#RunContext` API with a context already combined with standard signals.